### PR TITLE
Sync 3.x changes with master branch

### PR DESCRIPTION
**Pull request by jecihjoy**

### CODE REVIEW

File: .github/workflows/omoddeployment.yml
Comment (umair-fayaz) [inline .github/workflows/omoddeployment.yml:1]: Do we need this workflow in master? We already have a build/publish workflow that builds and publishes the omods to NEXUS

**Comment by binduak** [comment]
@jecihjoy do you want this to be merged to be merged to master branch. If yes, can you add more description to the PR along with a JIRA card. Also you can join one of the PAT calls to initiate discussion on the same.

**Comment by gsluthra** [comment]
This PR doesn't have any reference to a Bahmni JIRA issue number, and also doesn't mention any context on the huge number of changes. We cannot merge this PR until there is a JIRA ticket reference, with list of changes being sent over. Please create an issue in JIRA describing the fixes/enhancements and then mention them in this PR. Screenshots of changes will also be welcomed. thanks! https://bahmni.atlassian.net/jira/software/c/projects/AP/issues

**Comment by jecihjoy** [comment]
Closing this because I have PR'ed against wrong repo. Thanks.

**Pull request closed without merging**